test/unit/app/models/asset_test.rb

```diff
@@ -2,12 +2,12 @@
 
 class AssetTest < ActiveSupport::TestCase
   setup do
-    @attachment_data = create(:attachment_data_with_no_assets)
+    @image_data = create(:image_data_with_no_assets)
```

Comment:
It was easier to use image data to test the Asset class, as the Image storage does not require an attachable